### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by rajeshkumarhm** [pimcore/pimcore#15810] [Q&A / Support] [answered]
### Pimcore version

6.6.1

### Steps to reproduce

Click on any attribute label or area between the label and the value box, the value box gets selected automatically (text, select, multiselect, numeric etc.)

![Screenshot from 2023-08-24 11-17-10](https://github.com/pimcore/pimcore/assets/108528191/fbf95a0d-77dd-420a-958e-09937d9bbbed)

### Actual Behavior

By clicking on any attribute label or area between the label and the value box, the value box gets selected automatically (text, select, multiselect, numeric etc.). It is very annoying, specially in case of multiple multiselect dropdown. User need to click somewhere else in the screen to close it.

### Expected Behavior

By clicking on any attribute label or area between the label and the value box, nothing should be happened.

**Comment by dvesh3** [comment, marked as answer]
Hi @rajeshkumarhm Pimcore 6 is not supported anymore. Please see if this is still reproducible on latest version https://demo.pimcore.fun/admin

**Reply by rajeshkumarhm** [reply]
Hi Dvesh

It exists in Pimcore 11 as well. This might not be an issue from Pimcore side, but can we get some resolution on it.
